lib/pf/fingerbank.pm

```diff
@@ -100,14 +99,9 @@ sub _query {
     $mac = substr($mac, 0, 6);  # Only keep first 6 characters (OUI)
     $cached_args->{'mac'} = $mac;
 
-    return $cache->compute_with_undef($cached_args, 
-        sub {
-            $logger->debug("Fingerbank result not in cache (either not present or expired). Querying Fingerbank for result");
-            my $fingerbank = fingerbank::Query->new;
-            return $fingerbank->match($args);
-        },
-        {expires_in => FINGERBANK_CACHE_EXPIRE}
-    );
+    my $cache = pf::CHI->new( namespace => 'fingerbank', expires_in => FINGERBANK_CACHE_EXPIRE );
```

Comment:
We can't now as this will hit the maintenance branch and chi.conf will not be upgraded through it.

If we don't put it here and it's not in chi.conf then it will never expire.

We should create a chi.conf.defaults to handle these cases, as I agree with you, its not clean to have it here now